src/plugins/lua/luaplugin.cpp

```diff
@@ -655,7 +656,13 @@ void LuaWriter::writeMapObject(LuaTableWriter &writer,
         break;
     }
 
-    writeProperties(writer, mapObject->properties());
+    Properties props;
+    if (const ObjectTemplate *objectTemplate = mapObject->objectTemplate()) {
+        props.merge(objectTemplate->object()->properties());
```

Comment:
Here, we could just assign instead of calling `merge`, given that `props` is still empty anyway.

Please also remove the brackets, since there is only a single statement.